x-sounds-mac/Domain/XSound/XSoundsEndpointService.swift

```diff
@@ -0,0 +1,32 @@
+import Combine
+import Foundation
+import SwiftUI
+
+struct XSoundsEndpointService {
+    @AppStorage(AppStorageKeys.FMinstancePath.rawValue) static var instancePath = ""
+    
+    struct URLCreatingError: Error { }
+    
+    static func getSounds(webService: WebServiceProtocol) -> AnyPublisher<[XSound], Error> {
+        let urlString = instancePath + "x-sounds"
```

Comment:
My preference would be to do string interpolation instead of addition (my that's just my code style opinion)
```suggestion
        let urlString = "\(instancePath)x-sounds"
```

Reply:
Why are you not using `appendingPathComponent` fot this?